src/bundle/Resources/public/scss/fieldType/preview/_ezimage.scss

```diff
@@ -2,7 +2,43 @@
     display: flex;
 
     .ez-image__wrapper {
-        width: 50%;
+        flex-basis: 50%;
         margin-right: 2rem;
     }
+
+    .ez-field-preview {
+        &__image-meta {
+            flex-basis: 43%
+        }
+
+        &__actions-wrapper {
+            flex-basis: 7%
+        }
+
+        &__action {
+            .ez-icon {
+                transform: translate(-50%, 6px);
+            }
+
+            &--preview {
+                padding: 0 1.25rem;
+                border: none;
+                border-radius: 50%;
+                height: 40px;
+                width: 40px;
+                line-height: 40px;
+                cursor: pointer;
+                transition: background .3s $ez-admin-transition;
+                text-align: center;
+                display: flex;
+                margin-top: 0;
+                flex-wrap: nowrap;
```

Comment:
What I meant was, you can remove this if it's not needed. If it's ok with `display: block` please use block instead of flex. :)